bin/help.ml

```diff
@@ -45,10 +45,7 @@ let release =
         \        your source repository lints and that it builds in the \
          current build\n\
         \        environment and that the package tests pass.";
-      `Pre
-        "dune-release lint\n\
-         dune build # Check out the generated opam install file too\n\
```

Comment:
What does "Check out the generated opam install file too" mean? Should I keep that there?

Reply:
I think this refers to `dune-release distrib` that will run the lint on the archive, then checks that it builds, then checks the tests, it's not the doc of `dune-release lint`

Reply:
Isn't this the help page of `dune-release help release`?

Reply:
Oh yes right, well it looks like it's referring to the `<pkgname>.install` file that can be generated by dune build (didn't find any locally), but I think we can remove this line indeed!